docs/en/tutorials/how_to_install_mmdeploy_on_jetsons.md

```diff
@@ -26,98 +26,164 @@ Sometimes we just need to reboot the jetson device when it gets stuck in initial
 ```
 Or, it will throw the following error when we import torch in python:
 ```
 libmpi_cxx.so.20: cannot open shared object file: No such file or directory
 ```
+After that, download the PyTorch wheel for Jetson **specifically**. Click [here](https://forums.developer.nvidia.com/t/pytorch-for-jetson-version-1-10-now-available/72048) to get the wheel. MMDeploy is using `pytorch==1.8.0`.
+
+After the download finished, using cmd to install it.
+```shell
+pip3 install /your/path/dwonload/torch-1.8.0-cp36-cp36m-linux_aarch64.whl
+```
 
 ### Install torchvision
 We can't directly use `pip install torchvision` to install torchvision for Jetson Nano. But we can clone the repository from Github and build it locally. First we have to install some dependencies:
+```shell
+sudo apt-get install -y libjpeg-dev libpython3-dev libavcodec-dev libavformat-dev libswscale-dev
 ```
-sudo apt-get install libjpeg-dev libpython3-dev libavcodec-dev libavformat-dev libswscale-dev
-```
-Then just clone and compile the project:
-```
-git clone git@github.com:pytorch/vision.git
+Then just clone and compile the project, MMDeploy using `torchvision=0.9.0`:
+```shell
+git clone https://github.com/pytorch/vision.git
 cd vision
-git co tags/v0.7.0 -b vision07
+git checkout v0.9.0
 pip install -e .
 ```
 
+**Note:** If the `pip install` throws error: `Illegal instruction (core dumped)`, then add `export OPENBLAS_CORETYPE=ARMV8` in `~/.bashrc` and then `source ~/.bashrc && conda activate mmdeploy` will solve the problem.
+
 ### Install mmcv
 
 Install openssl first:
+```shell
+sudo apt-get install -y libssl-dev
 ```
-sudo apt-get install libssl-dev
+Then install it from source, MMDeploy using mmcv version is `1.4.0`:
+```shell
+git clone https://github.com/open-mmlab/mmcv.git
+cd mmcv
+git checkout v1.4.0
+MMCV_WITH_OPS=1 pip install -e .
 ```
-Then install it from source like `MMCV_WITH_OPS=1 pip install -e .`
 
 ### Update cmake
 
 We choose cmake version 20 as an example.
-```
+```shell
+sudo apt remove cmake
+sudo apt purge --auto-remove cmake
 sudo apt-get install -y libssl-dev
 wget https://github.com/Kitware/CMake/releases/download/v3.20.0/cmake-3.20.0.tar.gz
 tar -zxvf cmake-3.20.0.tar.gz
 cd cmake-3.20.0
 ./bootstrap
-make
+make -j$(nproc)
 sudo make install
 ```
 Then we can check the cmake version through:
-```
+```shell
 source ~/.bashrc
 cmake --version
 ```
 
-### Install mmdeploy
-Just follow the instruction [here](../build.md). If it throws `failed building wheel for numpy...ERROR: Failed to build one or more wheels` when installing `h5py`, try install `h5py` manually.
-```
-sudo apt-get install pkg-config libhdf5-100 libhdf5-dev
-pip install versioned-hdf5 --no-cache-dir
+### Install spdlog
+```shell
+sudo apt-get install libspdlog-dev
 ```
 
-Then install onnx manually. First, we have to install protobuf compiler:
-```
-sudo apt-get install libprotobuf-dev protobuf-compiler
+### Install onnxruntime
+```shell
+pip install onnxruntime==1.8.1
```

Comment:
what is the reason for installing onnxruntime?

Reply:
Is that we only need to install `onnx` ?

Reply:
As we assume users only care about TensorRT, onnx2trt requires onnx other than onnxruntime then.

Reply:
Done